checkov/terraform/graph_builder/graph_components/blocks.py

```diff
@@ -252,11 +252,13 @@ def to_dict(self) -> dict[str, Any]:
             'name': self.name,
             'path': self.path,
             'source': self.source,
-            'source_module': list(self.source_module)
+            'source_module': list(self.source_module),
+            'source_module_object': self.source_module_object
         }
 
     @staticmethod
     def from_dict(data: dict[str, Any]) -> TerraformBlock:
+        from checkov.terraform import TFModule
```

Comment:
can you use the full import here? Also did you check that it really cannot be imported at the global scope?